counsel.el

```diff
@@ -6182,6 +6182,15 @@ The buffers are those opened during a session of `counsel-switch-buffer'."
   "When non-nil, `counsel-switch-buffer' will preview virtual buffers."
   :type 'boolean)
 
+(defcustom counsel-switch-buffer-display-empty-buffer-when-no-preview nil
+  "How `counsel-switch-buffer' behaves when not displaying a preview.
+
+When non-nil, `counsel-switch-buffer' will display an empty buffer when
+the current candidate is not previewed (either a virtual buffer when `counsel-switch-buffer-preview-virtual-buffers' is nil or a large file).
```

Comment:
Please fill this paragraph to the usual 65-80 columns using <kbd>M-q</kbd> (`fill-paragraph`).